packages/driver/src/multi-domain/communicator.ts

```diff
@@ -149,6 +150,13 @@ export class SpecBridgeDomainCommunicator extends EventEmitter {
     }
   }
 
+  private resyncConfigToPrimary = () => {
```

Comment:
```suggestion
  private resyncConfigEnvToPrimary = () => {
```

Reply:
updated in b7258276e6134ccda3642b3936e8330370b26f03